test/test_prims.py

```diff
@@ -559,7 +559,8 @@ def fn1(x):
             self.assertFalse(fn0(x, y, 0.3, False))
 
             with torch.autocast("cuda"):
-                gm = make_fx(fn1, decomposition_table=torch._prims.context.nvfuser_decomp_table())(x)
+                gm = make_fx(fn1)(x)  # first call to get autocast context calls recorded
+            gm = make_fx(gm)(x)  # second call to get autocast context calls converted to nvprims
```

Comment:
lol, I get what we are trying to do here.

The comment here is a little confusing though. IIUC, `gm = make_fx(fn1)(x)` already does a portion of the fx graph converted to nvprims.

Reply:
The first make_fx call does have some computations converted to nvprims but not the autocast-related calls. Can you please suggest an alternative message?

Reply:
I think we can comment focusing on the limitation here. `first trace records autocast c++ calls directly and doesn't have the chance to translate to nvprim through decomposition map in "TorchRefsNvfuserCapabilityMode"`. Feel free to re-work the wording here.